recipe/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "tenseal" %}
+{% set version = "0.3.14" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenMined/TenSEAL/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d3c086836200c45ad6b9ac0a149b535aa3bc5dd5dac8d1bf5f8fe1706e234623
+  patches:
+    - patches/0001-unvendor-deps.patch
+    - patches/0002-Fix-windows-build.patch  # [win]
+
+build:
+  number: 0
+  script_env:
+    - VERBOSE=1
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -v
+  missing_dso_whitelist:
+    - '$RPATH/ld64.so.1'  # [s390x]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - patch  # [not win]
+    - m2-patch  # [win]
+    - make  # [not win]
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - seal 4.1.1
+    - zlib {{ zlib }}
+    - libprotobuf {{ libprotobuf }}
+    - pybind11 2.10
+    - xtensor 0.23.5
+  run:
+    - python
+    - zlib  # pin through run_exports
+    - libprotobuf  # pin through run_exports
+    - numpy
+
+test:
+  source_files:
+    - tests/python
+    - tests/__init__.py
```

```suggestion
    - tests/__init__.py
    - pytest.ini
```
(Required for pytest to recognize the slow marker.

---

It's not necessary and it just removes the pytest warning. But I can add it to remove the warning.